frontend/src/container/OnboardingContainer/OnboardingContainer.tsx

```diff
@@ -279,58 +285,110 @@ export default function Onboarding(): JSX.Element {
 	}, []);
 
+	const [form] = Form.useForm<InviteMemberFormValues>();
+	const [
+		isInviteTeamMemberModalOpen,
+		setIsInviteTeamMemberModalOpen,
+	] = useState<boolean>(false);
+
+	const toggleModal = useCallback(
+		(value: boolean): void => {
+			setIsInviteTeamMemberModalOpen(value);
+			if (!value) {
+				form.resetFields();
+			}
+		},
+		[form],
+	);
+
 	return (
 		<div className={cx('container', isDarkMode ? 'darkMode' : 'lightMode')}>
 			{activeStep === 1 && (
-				<>
-					<FullScreenHeader />
-					<div className="onboardingHeader">
-						<h1> Select a use-case to get started</h1>
-					</div>
-					<div className="modulesContainer">
-						<div className="moduleContainerRowStyles">
-							{Object.keys(ModulesMap).map((module) => {
-								const selectedUseCase = (useCases as any)[module];
-
-								return (
-									<Card
-										className={cx(
-											'moduleStyles',
-											selectedModule.id === selectedUseCase.id ? 'selected' : '',
-										)}
-										style={{
-											backgroundColor: isDarkMode ? '#000' : '#FFF',
-										}}
-										key={selectedUseCase.id}
-										onClick={(): void => handleModuleSelect(selectedUseCase)}
-									>
-										<Typography.Title
-											className="moduleTitleStyle"
-											level={4}
+				<div className="onboarding-page">
+					<div>
+						<div
+							onClick={(): void => {
+								logEvent('Onboarding V2: Skip Button Clicked', {});
+								history.push('/');
+							}}
+							className="skip-to-console"
+						>
+							{t('skip')}
+						</div>
+						<FullScreenHeader />
+						<div className="onboardingHeader">
+							<h1>{t('select_use_case')}</h1>
+						</div>
+						<div className="modulesContainer">
+							<div className="moduleContainerRowStyles">
+								{Object.keys(ModulesMap).map((module) => {
+									const selectedUseCase = (useCases as any)[module];
+
+									return (
+										<Card
+											className={cx(
+												'moduleStyles',
+												selectedModule.id === selectedUseCase.id ? 'selected' : '',
+											)}
 											style={{
-												borderBottom: isDarkMode ? '1px solid #303030' : '1px solid #ddd',
-												backgroundColor: isDarkMode ? '#141414' : '#FFF',
+												backgroundColor: isDarkMode ? '#000' : '#FFF',
 											}}
+											key={selectedUseCase.id}
+											onClick={(): void => handleModuleSelect(selectedUseCase)}
 										>
-											{selectedUseCase.title}
-										</Typography.Title>
-										<Typography.Paragraph
-											className="moduleDesc"
-											style={{ backgroundColor: isDarkMode ? '#000' : '#FFF' }}
-										>
-											{selectedUseCase.desc}
-										</Typography.Paragraph>
-									</Card>
-								);
-							})}
+											<Typography.Title
+												className="moduleTitleStyle"
+												level={4}
+												style={{
+													borderBottom: isDarkMode ? '1px solid #303030' : '1px solid #ddd',
```

Comment:
Get rid of inline styles.

Reply:
These were existing codes, just added a wrapper div, hence showing as dif, but will check

Reply:
done